engine/gui/src/gui.cpp

```diff
@@ -1805,17 +1805,14 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
                     lua_newtable(L);
 
                     if (ia->m_IsGamepad) {
-                        lua_pushliteral(L, "gamepad");
                         lua_pushnumber(L, ia->m_GamepadIndex);
-                        lua_settable(L, -3);
+                        lua_setfield(L, -2, "gamepad");
```

Comment:
The missing "lu_settable" messed up the lua stack, and caused a panic that shut the application down.

I prefer "lua_setfield" for readability and it helps keep the "key-value" approach simpler.

Reply:
(I'm suspecting a bad merge somewhere)